Sources/macros.js

```diff
@@ -174,6 +174,20 @@ function safeArrays(model) {
   });
 }
 
+function isTypedArray(value) {
+  return (
```

Comment:
If we replace this with `Object.values(TYPED_ARRAYS).some((ctor) => value instanceof ctor)`, we can handle `Big[I|Ui]nt64Array` as well.